src/app/auth/components/login/login.component.ts

```diff
@@ -57,6 +59,21 @@ export class LoginComponent implements OnInit {
           this.router.navigate(['/products']);
         }
     });
+    // Error handing
+    this.store.select( states => {
```

Comment:
Using "store" for error handling, error message is coming from login API (has some validation), also added action and reducer for error handling.